tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_deps.sh

```diff
@@ -15,4 +15,15 @@ do
   ${PYTHON_EXE} -m pip install -r requirements.txt
 done
 
-
+# No release binary for ccache aarch64, so we need to build it from source.
+if ! [ -x "$(command -v ccache)" ]; then
+    git clone --recursive --branch v4.7.4 https://github.com/ccache/ccache
```

Comment:
Can you use a tarball or zip file instead? Like what we have in deps.txt. Otherwise, when you clone a repo, you always download all branches of it.